src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1326,9 +1327,32 @@ private static string GetParamDefaultValueAsString(object? value, string objectT
             {
                 case null:
                     return $"default({objectTypeAsStr})";
-                case bool boolVal:
-                    return ToCSharpKeyword(boolVal);
+                case bool @bool:
+                    return ToCSharpKeyword(@bool);
+                case string @string:
+                    return @$"""{@string}""";
```

Comment:
Would this produce the correct results if the string includes escape sequences?
Perhaps testing with a more complex input (e.g. `"\n\r⁉️\'\"\u200D"`) might illustrate the expected results.

Similar question likely applies to the `case char` below too.

Reply:
Thanks, the previous approach was insufficient. Next commit addresses the issues with escaping.